src/main/java/pages/IssuePage.java

```diff
@@ -0,0 +1,63 @@
+package pages;
+
+import static com.codeborne.selenide.Condition.text;
+import static com.codeborne.selenide.Selectors.byId;
+import static com.codeborne.selenide.Selectors.byXpath;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+import static com.codeborne.selenide.Selenide.open;
+
+public class IssuePage {
+
+    String commentButtonID = "footer-comment-button";

```

Comment:
I sugges rewrite selectors from format:
String priorityFieldID = "priority-field";
 
public void clickPriorityElement(){ 
$(byId(priorityElementID)).click();
}

to:

public void clickPriorityElement(){ $(By.id("priority-field")).click();}

Reply:
In case if selector used only once this is good suggestion.